controllers/userStatus.js

```diff
@@ -1,8 +1,8 @@
-const { Forbidden, NotFound } = require("http-errors");
-const { fetchUser } = require("../models/users");
-const userStatusModel = require("../models/userStatus");
 const { getUserIdBasedOnRoute } = require("../utils/userStatus");
 const { INTERNAL_SERVER_ERROR } = require("../constants/errorMessages");
+const dataAccess = require("../services/dataAccessLayer");
+const { Forbidden, NotFound } = require("http-errors");
```

Comment:
NPM imports (external imports) in general ideally should be at top before the local imports.